app/forms/csv_upload_form.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CsvUploadForm
+  include ActiveModel::Model
+
+  attr_reader :keyword_ids
+
+  validates :file, presence: true, csv_keyword_file: true
+
+  def save(file)
+    @file = file
+    return false unless valid?
+
+    begin
```

Comment:
Just a small thing, we can omit the `begin` here, that is not a big topic @Goose97 :)

Reply:
Removed the rescue block here cd0402d71dd82d0fbabe17c4271371a605df2f42